app/models/tag.rb

```diff
@@ -2,4 +2,8 @@ class Tag < ActiveRecord::Base
   has_many :taggings
   has_many :jobs, through: :taggings
 
+
+  def tag_count
+    Tagging.where(tag_id: self.id).map.count
```

Comment:
Keep in mind the law of Demeter, AR allows you to do this without reaching through the join table.